src/sign_workflow/sign_args.py

```diff
@@ -28,6 +30,7 @@ def __init__(self) -> None:
         parser.add_argument("--type", help="Artifact type")
         parser.add_argument("--sigtype", choices=self.ACCEPTED_SIGNATURE_FILE_TYPES, help="Type of signature file.", default=".sig")
         parser.add_argument("--platform", choices=self.ACCEPTED_PLATFORM, help="Distribution platform.", default="linux")
+        parser.add_argument("-r", "--allow_output_override", choices=self.ACCEPTED_OVERRIDE_VALUES, help="Boolean to allow output file to override existing file", default="False")
```

Comment:
Suggest using dashes instead of underscore for these arguments.

Reply:
The argument can be dash while the internal var name can be underscore.

Reply:
And I would remove the short version of the param as it is confusing.
Or add short term for all existing ones.

Reply:
I had dashes before. Had to change to underscore because of https://github.com/opensearch-project/opensearch-build-libraries/blob/main/vars/signArtifacts.groovy#L162. Groovy does not allow `-` so libtesters were giving issues. Will see if I can acomodate something else

Reply:
Sure thanks.